Directory.Build.targets

```diff
@@ -42,7 +40,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="LeanCode.CodeAnalysis" Version="$(CoreLibVersion)" PrivateAssets="all" />
+    <PackageReference Include="LeanCode.CodeAnalysis" Version="6.0.1583" PrivateAssets="all" />
+    <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.435" PrivateAssets="all" />
```

Comment:
Let's bump the StyleCop version in CoreLib _then_ let's use it here.

Reply:
It is new enough in CoreLib, but 6.0.1583 is the newest CoreLib available on NuGet. This is just me not wanting to add extra feeds just to be able to build the generator.

Reply:
Then let's publish the version on NuGet. :P

I really don't want to have stray overrides here and there.

Reply:
What if we _removed_ `LeanCode.CodeAnalysis` instead? This would save us weird cyclical dependencies between repositories, which, while not currently problematic, would probably cause anyone reading this for the first time to be quite confused.

Although this way we lose(?) our custom ruleset file, which is probably not optimal either…

Reply:
The main addition of LeanCode.CondeAnalysis is the shared ruleset - that is the reason we use it across all the projects.

I would prefer moving the `LeanCode.CodeAnalysis` to a separate repo than removing it.

Reply:
:cold_sweat:

So much to do just to remove one `SuppressMessage` :slightly_frowning_face: